File: server/Models/Post.cs
namespace mobSite.Models;
public class Post
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string Title { get; set; }
    public string Body { get; set; }
    public int ProjectId { get; set; }
    public int TierId { get; set; }
    public string CreatorId { get; set; }
    public Account Creator { get; set; }
}

File: client/src/models/Post.js
export class Post{
    constructor(data){
        this.id = data.id
        this.title = data.title
        this.body = data.body
         this.shortBody = data.body.slice(0, 400)
        this.img = data.img
        this.tierId= data.tierId
        this.projectId = data.projectId
        this.creator = data.creator
        this.creatorId = data.creatorId
    }
    get shortDescription() {
        return this.body.slice(0, 50)
    }
}